src/lib.rs

```diff
@@ -89,11 +108,14 @@ impl_nb_bits!(true, f16);
 //                     }
 //                     #[cfg(target_arch = "aarch64")]
 //                     {
-//                         // TODO: support aarch64
+//                          if *NEON_DETECTED & (<$t>::NB_BITS < 64) {
+//                              // We miss some NEON instructions for 64-bit numbers
+//                              return unsafe { NEON::argminmax(self) }
+//                          }
 //                     }
 //                     #[cfg(target_arch = "arm")]
 //                     {
-//                         if *NEON_DETECTED & (<$t>::NB_BITS < 32) {
+//                         if *NEON_DETECTED & (<$t>::NB_BITS < 64) {
 //                             // TODO: requires v7?
 //                             // We miss some NEON instructions for 64-bit numbers
 //                             return unsafe { NEON::argminmax(self) }
```

Comment:
Lots of unused code here - we could probably remove this section or at least explain what it does.

Reply:
Yeah is something I explored in the past - but did not find the time to properly assess the performance impact. I recall observing very minimal / no impact